### PR TITLE
cognitive - regenerate embedded sdk

### PR DESCRIPTION
**Pull request by mbfrahry**
image

The failing test is transient

### CODE REVIEW

**Review by katbyte — APPROVED** [review]
LGTM 🍿

File: internal/services/cognitive/sdk/2021-04-30/cognitiveservicesaccounts/constants.go [hunk 1]
```diff
@@ -9,27 +11,123 @@ const (
...
+	return []string{
+		"Microsoft.CognitiveServices",
+		"Microsoft.KeyVault",
+	}
+}
+
+func parseKeySource(input string) (*KeySource, error) {
+	vals := map[string]KeySource{
+		"microsoftpointcognitiveservices": "Microsoft.CognitiveServices",
+		"microsoftpointkeyvault":          "Microsoft.KeyVault",
```
Comment (tombuildsstuff) [inline internal/services/cognitive/sdk/2021-04-30/cognitiveservicesaccounts/constants.go:88]: that's a bug here

Comment (tombuildsstuff) [reply internal/services/cognitive/sdk/2021-04-30/cognitiveservicesaccounts/constants.go]: regenerated

**Review by jackofallops — APPROVED** [review]
LGTM 👍

**Pull request merged**